website/docs/cloud-docs/run/states.mdx

```diff
@@ -136,7 +136,27 @@ _Leaving this stage:_
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-## 8. The Apply Stage
+## 8. The Pre-Apply Stage
+
+The pre-apply phase only occurs if there are enabled [run tasks](/cloud-docs/workspaces/settings/run-tasks) in the workspace that are configured to begin before Terraform creates the apply. Terraform Cloud sends information about the run to the configured external system and waits for a `passed` or `failed` response to determine whether the run can continue. The information sent to the external system includes the configuration version of the run.
+
+-> **Note:** Pre-apply run tasks are in beta.
+
+Only runs that are confirmed can enter this phase.
+
+_States in this stage:_
+
+- **Pre-apply running:** Terraform Cloud is waiting for a response from the configured external system(s).
+  - External systems must respond initially with a `200 OK` acknowledging the request is in progress. After that, they have 10 minutes to return a status of `passed`, `running`, or `failed`. If the timeout expires, Terraform Cloud assumes that the run tasks is in the `failed` status.
+
+_Leaving this stage:_
+
+- If any mandatory tasks failed, the run skips to completion.
+- If any advisory tasks failed, the run proceeds to the **Applying** state, with a visible warning regarding the failed task.
+- If a combination of mandatory and advisory tasks are configured for a single run, Terraform takes the most restrictive action. For example, the run fails if there are two advisory tasks that succeed and one mandatory task that fails.
```

Comment:
```suggestion
- If a single run has a combination of mandatory and advisory tasks, Terraform takes the most restrictive action. For example, the run fails if there are two advisory tasks that succeed and one mandatory task that fails.
```

Suggest changing "a combination of mandatory and advisory tasks are configured for a single run" to "a single run has a combination of mandatory and advisory tasks" to avoid passive voice style issues and to use fewer words.